cmd/ecs.go

```diff
@@ -298,7 +298,7 @@ func runECSServiceLsCmd(cmd *cobra.Command, args []string) error {
 
 func newECSServiceUpdateCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "update CLUSTER SERVICE",
+		Use:   "update CLUSTER",
```

Comment:
This fix is not directly related, but I noticed incorrect help message and fixed it.